modules/ui.py

```diff
@@ -904,6 +906,7 @@ def select_img2img_tab(tab):
 
             img2img_prompt.submit(**img2img_args)
             submit.click(**img2img_args)
+            detect_image_size_btn.click(lambda i, w, h : i.size if i is not None else (w, h), inputs=[init_img, width, height], outputs=[width, height])

```

Comment:
Doing this with `init_img.change()` instead would make it automatic.

And you've only done it for the `img2img` tab. The others have their own image inputs.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/771ea212de13711b494b082d8e94e79b17ac9d08/modules/ui.py#L686-L714